src/design-system/components/Box/Box.tsx

```diff
@@ -284,7 +283,7 @@ export const Box = forwardRef(function Box(
 
     // We flatten the styles array in case it's passed to Animated.View.
     // This won't be needed with v2.3+ of react-native-reanimated.
-    return Component === View ? stylesArray : flatten(stylesArray);
+    return Component === View ? stylesArray : stylesArray.flatMap(key => key);
```

Comment:
Nit: Could this just use `flat` instead?

```suggestion
    return Component === View ? stylesArray : stylesArray.flat();
```

Reply:
yea, I will replace it